pkg/manifest/manifest_loader.go

```diff
@@ -149,36 +150,102 @@ func LoadManifest(context *LoaderContext) (Manifest, []error) {
 
 	relativeManifestPath := filepath.Base(manifestPath)
 
+	var errs []error
+
+	// projects
 	projectDefinitions, projectErrors := toProjectDefinitions(&projectLoaderContext{
 		fs:           workingDirFs,
 		manifestPath: relativeManifestPath,
 	}, manifestYAML.Projects)
-
-	var errs []error
 	if projectErrors != nil {
 		errs = append(errs, projectErrors...)
 	} else if len(projectDefinitions) == 0 {
 		errs = append(errs, newManifestLoaderError(context.ManifestPath, "no projects defined in manifest"))
 	}
 
+	// environments
 	environmentDefinitions, manifestErrors := toEnvironments(context, manifestYAML.EnvironmentGroups)
-
 	if manifestErrors != nil {
 		errs = append(errs, manifestErrors...)
 	} else if len(environmentDefinitions) == 0 {
 		errs = append(errs, newManifestLoaderError(context.ManifestPath, "no environments defined in manifest"))
 	}
 
+	// accounts
+	accounts, accErrs := convertAccounts(context, manifestYAML.Accounts)
```

Comment:
nit: `toAccounts` or also rename `toEnvironments` to `convert` - it's odd that functions that do the same operation on different types have different naming patterns

Reply:
Moved everything to *parse*
Its a difficult topic, as we mix two things in this package:
* Reading
* Writing

`convert` would not state from what to what
`to` is just a non-telling prefix, also does not state from what to what
`parse` is also not perfect, as we're not really parsing anything. However, since we do validation and conversion, I think it's the most fitting.

According to Bing Chat, a good name would be `transform`, but same problem as `convert` and `to`.

Reply:
I thought about that but then decided against it in this PR to keep it simple. The structure we'd need to introduce is the following:

```
manifest/
  .                    # in memory definitions
  loading             
  writing
  internal/persistence # persistence definitions
```

I can try it on a follow-up if you agree that it makes sense and is not too much overhead.

Reply:
I think the change/effort would be mostly shuffling files around? IMO worth at least trying to see if feels any more organised :) 

And yes, 100% meant as an optional followup, thus already approved (and merged as I see 👍 )